core/controllers/skill_mastery.py

```diff
@@ -33,6 +34,25 @@ class SkillMasteryDataHandler(base.BaseHandler):
     """
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {
+            'comma_separated_exp_ids': {
+                'schema': {
+                    'type': 'basestring'
+                }
+            }
+        },
+        'PUT': {
```

Comment:
@vojtechjelinek Shouldn't `def put` use `self.normalized_payload`? Also, is there a corresponding `normalized_request` for `get`?

Reply:
Ah, yes, use `self.normalized_payload` instead of `self.payload` everywhere.